src/templates/static.tsx

```diff
@@ -189,8 +189,6 @@ const StaticPage = ({
   const { editContentUrl } = siteData.siteMetadata || {}
   const absoluteEditPath = `${editContentUrl}${relativePath}`
 
-  const slug = mdx.fields?.slug || ""
```

Comment:
This `slug` is incorrrect. This `slug` is always the `/en/` version. To fix this, now we are getting it from the `pageContext`.